src/main/java/uk/nhs/hee/tis/trainee/reference/api/CollegeResource.java

```diff
@@ -0,0 +1,37 @@
+package uk.nhs.hee.tis.trainee.reference.api;
+
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.nhs.hee.tis.trainee.reference.dto.CollegeDto;
+import uk.nhs.hee.tis.trainee.reference.mapper.CollegeMapper;
+import uk.nhs.hee.tis.trainee.reference.model.College;
+import uk.nhs.hee.tis.trainee.reference.service.CollegeService;
+
+
+@RestController
+@RequestMapping("/api")
+public class CollegeResource {
+
+  private static final Logger log = LoggerFactory.getLogger(CollegeResource.class);
+  @Autowired
```

Comment:
Prefer constructor injection over annotations.

Reply:
All the annotations has been modified to constructor injection.